docs/exportCypher.adoc

```diff
@@ -14,6 +14,8 @@ It is possible to choose between three export formats:
 * `cypher-shell`: for Cypher shell
 * `plain`: doesn't output begin / commit / await just plain Cypher
 
+You can also use the Optimizations like: `{useOptimizations: true}`. (default: false)
```

Comment:
can we use a more concrete name, like `unwindBatch:true/false`

Reply:
oh and we should default it to true

Reply:
or
useOptimization: null or 'none'
useOptimization: 'unwindBatch'
useOptimization: 'unwindBatchParams'

or if we want to have multiple flags later ...

useOptimization: ['unwindBatch','param']

Reply:
`useOptimizations: { unwindBatch: true, batchSize: 100, params: true}`